lib/core/main.js

```diff
@@ -222,7 +223,7 @@ async function image(query, options = {}) {
 async function getTopNews(language = 'en', region = 'US') {
   const url = Constants.URLS.GOOGLE_NEWS + `topstories?tab=in&hl=${language.toLocaleLowerCase()}-${region.toLocaleUpperCase()}&gl=${region.toLocaleUpperCase()}&ceid=${region.toLocaleUpperCase()}:${language.toLocaleLowerCase()}`;
 
-  const response = await Axios.get(url, { headers: Utils.getHeaders({ mobile: true }) }).catch((err) => err);
+  const response = await Axios.get(url, { headers: Utils.getHeaders({ mobile: true }), ...options.axios_config }).catch((err) => err);
```

Comment:
The `options` param doesn't exist in this function though, other than that everything looks good!